assets/src/features/home-page/components/HomePageLayout.tsx

```diff
@@ -0,0 +1,29 @@
+import clsx from "clsx";
+import React, { PropsWithChildren } from "react";
+
+import Navbar from "./Navbar";
+
+const HomePageLayout: React.FC<PropsWithChildren> = ({ children }) => {
+  return (
+    <div
+      className={clsx(
+        "h-screen w-full bg-brand-sea-blue-200 text-brand-dark-blue-500",
+        "font-rocGrotesk",
+        "flex flex-col items-center p-4",
+        "relative"
+      )}
+      style={{
+        backgroundImage: "url('/images/backgroundLeft.png'), url('/images/backgroundRight.png')",
```

Comment:
It seems that tailwind doesn't support having multiple backgrounds. It would require using syntax for arbitrary values which is much less legible in this case